database/database.go

```diff
@@ -56,6 +52,7 @@ type (
 	DB struct {
 		staticDB    *mongo.Database
 		staticUsers *mongo.Collection
+		staticDep   lib.Dependencies
```

Comment:
Why are all these fields listed as `static`. The `DB` struct doesn't have a mutex

Reply:
Don't we only do that for structs that have mutexes? I don't mind keeping it though. I thought we marked as static only when there's a mutex, so people trying to access the field know it's safe, which indeed can only be if it never changes if the struct has a mutex.

Reply:
It just means that the field won't be touched after initializing it. Which implies that it's safe to access without locking. I don't really care that much whether we have static fields within structs that don't have a mutex but it might still be a useful indication. Especially if we ever add locking to a struct later on.